openmetadata-service/src/main/resources/elasticsearch/en/table_index_mapping.json

```diff
@@ -615,11 +616,67 @@
           }
         }
       },
-      "lineage": {
-        "type" : "object"
+      "upstreamLineage": {
```

Comment:
@harshach this needs to be added for other entities as well,  also dynamic : false might cause issues , since pipeline whole object is being put or we need to cleanup  below method in LineageRepository

  public static void addPipelineDetails(EsLineageData lineageData, EntityReference pipelineRef) {
    if (nullOrEmpty(pipelineRef)) {
      lineageData.setPipeline(null);
    } else {
      Pair<String, Map<String, Object>> pipelineOrStoredProcedure =
          getPipelineOrStoredProcedure(pipelineRef, List.of("changeDescription"));
      lineageData.setPipelineEntityType(pipelineOrStoredProcedure.getLeft());
      lineageData.setPipeline(pipelineOrStoredProcedure.getRight());
    }
  }